std/algorithm/mutation.d

```diff
@@ -755,6 +756,21 @@ void initializeAll(Range)(Range range)
     range[] = T.init;
 }
 
+///
```

Comment:
This unittest isn't showing up in the generated docs, because the second overload of `initializeAll` is preceded by `// ditto` (missing one `/`), instead of `/// ditto`, so ddoc fails to recognize it as part of the previous declaration.